core/test/matrix/sparsity_csr.cpp

```diff
@@ -99,10 +99,8 @@ class SparsityCsr : public ::testing::Test {
         ASSERT_EQ(m->get_size(), gko::dim<2>(0, 0));
         ASSERT_EQ(m->get_num_nonzeros(), 0);
         ASSERT_EQ(m->get_const_col_idxs(), nullptr);
-        ASSERT_EQ(m->get_const_row_ptrs(), nullptr);
         ASSERT_EQ(m->get_const_value(), nullptr);
         ASSERT_EQ(m->get_col_idxs(), nullptr);
-        ASSERT_EQ(m->get_row_ptrs(), nullptr);
```

Comment:
Same here: Since you actually want to allocate every time, I would change it to:
```
ASSERT_NE(m->get_const_row_ptrs(), nullptr);
ASSERT_NE(m->get_row_ptrs(), nullptr);
```